[2024-10-16T09:56:07.027+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-10-16T09:56:07.048+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-10-16T09:56:07.050+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:56:07.051+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:56:07.052+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:56:07.078+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-24 00:00:00+00:00
[2024-10-16T09:56:07.085+0000] {standard_task_runner.py:55} INFO - Started process 8994 to run task
[2024-10-16T09:56:07.091+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '1849', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp9znsqbfx']
[2024-10-16T09:56:07.092+0000] {standard_task_runner.py:83} INFO - Job 1849: Subtask insert_rate
[2024-10-16T09:56:07.225+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:56:07.316+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:56:07.410+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-24T00:00:00+00:00
[2024-10-16T09:56:07.412+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 466.74597117, '2024-06-24')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:56:07.426+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:56:07.789+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:56:07.805+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 466.74597117, '2024-06-24')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:56:07.808+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:56:07.830+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240624T000000, start_date=20241016T095607, end_date=20241016T095607
[2024-10-16T09:56:08.015+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:56:08.047+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:06:09.503+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-10-16T10:06:09.516+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-10-16T10:06:09.516+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:09.517+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:06:09.518+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:06:09.538+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-06-24 00:00:00+00:00
[2024-10-16T10:06:09.544+0000] {standard_task_runner.py:55} INFO - Started process 11555 to run task
[2024-10-16T10:06:09.551+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '2580', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmp5a_i2n4l']
[2024-10-16T10:06:09.552+0000] {standard_task_runner.py:83} INFO - Job 2580: Subtask insert_rate
[2024-10-16T10:06:09.650+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-06-24T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:06:09.696+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:06:09.753+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-06-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-24T00:00:00+00:00
[2024-10-16T10:06:09.754+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 466.74597117, '2024-06-24')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:06:09.766+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:06:10.014+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:06:10.023+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 466.74597117, '2024-06-24')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:06:10.026+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:06:10.041+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240624T000000, start_date=20241016T100609, end_date=20241016T100610
[2024-10-16T10:06:10.099+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:06:10.123+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
